sdktests/php_events_all.go

```diff
@@ -34,7 +34,14 @@ func doPHPEventRequestTests(t *ldtest.T) {
 		}))
 
 	eventTests.RequestMethodAndHeaders(t, sdkKey,
-		Header("X-LaunchDarkly-Event-Schema").Should(m.Equal(phpEventSchema)))
```

Comment:
In this case, should `phpEventSchema` be renamed something like `php(Deprecated|Legacy|Whatever)EventSchema` to indicate that it's no longer the latest and greatest?

Reply:
Done!